### DISCUSSION TITLE
HTML injection via changelogs

### DISCUSSION THREAD

**Discussion by jpgoelz** [renovatebot/renovate#27797] [Request Help]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

37.231.2

### Please tell us more about your question or problem

I just noticed, that using HTML in Changelogs can lead to the HTML being parsed in the Renovate-Changelog message:

I don't know if this poses any risks. But maybe it is better to replace `<` and `>` with `&lt;` and `&gt;`?

Here is an example:

![image](https://github.com/renovatebot/renovate/assets/32487587/2369fd36-3b52-4183-8c00-316edc0334af)

![image](https://github.com/renovatebot/renovate/assets/32487587/b5b22b65-1db8-4d23-85c0-e3ee176e36ec)

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
It does not pose a security risk because the code platforms already defend against users creating issues or PRs maliciously. Please report any security concerns privately in future and not in public (same goes for any open source project).

If you have any specific examples you think could be escaped, please suggest a feature.

**Reply by jpgoelz** [reply]
Very good point!
